packages/plugins/src/hardhat/tasks.ts

```diff
@@ -210,10 +213,37 @@ task(TASK_CHUGSPLASH_DEPLOY)
     'ipfsUrl',
     'Optional IPFS gateway URL for publishing ChugSplash projects to IPFS.'
   )
+  .addFlag('isUpgrade', 'Perform an upgrade on the chosen project')
```

Comment:
I actually don't think this flag should be shown to the user. I.e. ChugSplash should automatically determine if it's a deployment/upgrade